ddl/placement_policy_test.go

```diff
@@ -34,73 +34,63 @@ import (
 	"github.com/pingcap/tidb/parser/model"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/store/gcworker"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/util/testutil"
```

Comment:
You can replace `testutil.RowsWithSep` to `testkit.RowsWithSep` (new testkit has implemented this method) and remove this dep. I think `util/testutil` is also dep to be deprecated.

Reply:
have removed.